Sources/Widgets/Core/WidgetManager/index.js

```diff
@@ -481,11 +499,19 @@ function vtkWidgetManager(publicAPI, model) {
       return model.previousSelectedData;
     }
 
-    if (!propsWeakMap.has(prop)) {
-      return {};
+    let widget = null;
+    let representation = null;
+
+    if (propsWeakMap.has(prop)) {
+      const props = propsWeakMap.get(prop);
+      widget = props.widget;
+      representation = props.representation;
+    } else {
+      const selectionProps = model.selections[0].getProperties();
```

Comment:
Can `model.selections` have zero length here, or are we always guaranteed to have at least one element?

Reply:
we are sure model.selections is not empty. First thing checked in `getSelectedData()`